database/aoai-proxy.sql

```diff
@@ -173,6 +173,55 @@ $$;
 
 ALTER FUNCTION aoai.add_event_attendee(p_user_id character varying, p_event_id character varying) OWNER TO admin;
 
+--
+-- Name: aoai_add_metric(uuid, uuid, uuid, integer); Type: FUNCTION; Schema: aoai; Owner: admin
+--
+
+CREATE FUNCTION aoai.aoai_add_metric(p_api_key uuid, p_event_id uuid, p_catalog_id uuid, p_daily_request_cap integer) RETURNS bigint
```

Comment:
```suggestion
CREATE FUNCTION aoai.add_metric(p_api_key uuid, p_event_id uuid, p_catalog_id uuid, p_daily_request_cap integer) RETURNS bigint
```

Don't need the `aoai_` prefix on the function name